release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -124,6 +124,26 @@ You can now move etcd from a root volume (Cinder) to a dedicated ephemeral local
 [id="ocp-4-15-web-console"]
 === Web console
 
+[id="ocp-4-15-administrator-perspective"]
+==== Administrator perspective
+
+This release introduces the following updates to the *Administrator* perspective of the web console:
+* Enable and disable the tailing to Pod log viewer to minimize load time.
+* View recomended values for `VerticalPodAutoscaler` on the *Deployment* page.
```

Comment:
@opayne1 if these 2 update points are expected to list in 2 lines? from review [doc](https://70760--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes#ocp-4-14-administrator-perspective),  I see they're still shown in one line with * in texts.

Reply:
Yes, they are intended to be 2 bullets. I have fixed! Thanks for catching that.